templates/head.pug

```diff
@@ -7,7 +9,10 @@ mixin head()
   title= siteTitle
   link(href="/css/sticky-footer-navbar.css", rel="stylesheet")
   link(href="/css/cytube.css", rel="stylesheet")
-  link(id="usertheme", href="/css/themes/slate.css", rel="stylesheet")
+  link(id="usertheme", href=DEFAULT_THEME, rel="stylesheet")
+
+  script(type="text/javascript").
+    var DEFAULT_THEME = '#{DEFAULT_THEME}';
```

Comment:
Didn't you mention in your Pug upgrade PR that interpolation is no longer supported?  Or is it still for JS string literals?